[2022-02-15 03:06:11,140] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.dim_customers manual__2022-02-15T03:04:57.520087+00:00 [queued]>
[2022-02-15 03:06:11,152] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.dim_customers manual__2022-02-15T03:04:57.520087+00:00 [queued]>
[2022-02-15 03:06:11,153] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:06:11,154] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:06:11,155] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:06:11,170] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.dim_customers> on 2022-02-15 03:04:57.520087+00:00
[2022-02-15 03:06:11,176] {standard_task_runner.py:52} INFO - Started process 657 to run task
[2022-02-15 03:06:11,191] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.dim_customers', 'manual__2022-02-15T03:04:57.520087+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmp_mzbh0s1', '--error-file', '/tmp/tmph9akxyu0']
[2022-02-15 03:06:11,192] {standard_task_runner.py:77} INFO - Job 99: Subtask model.my_new_project.dim_customers
[2022-02-15 03:06:11,303] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.dim_customers manual__2022-02-15T03:04:57.520087+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:06:11,444] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.dim_customers
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:04:57.520087+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:04:57.520087+00:00
[2022-02-15 03:06:11,446] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:06:11,449] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models dim_customers             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 03:06:11,472] {subprocess.py:85} INFO - Output:
[2022-02-15 03:06:19,431] {subprocess.py:89} INFO - 03:06:19  Running with dbt=1.0.1
[2022-02-15 03:06:19,872] {subprocess.py:89} INFO - 03:06:19  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:06:19,888] {subprocess.py:89} INFO - 03:06:19
[2022-02-15 03:06:23,677] {subprocess.py:89} INFO - 03:06:23  Concurrency: 4 threads (target='dev')
[2022-02-15 03:06:23,679] {subprocess.py:89} INFO - 03:06:23
[2022-02-15 03:06:23,692] {subprocess.py:89} INFO - 03:06:23  1 of 1 START table model dbt_bytecode_training.dim_customers.................... [RUN]
[2022-02-15 03:06:27,492] {subprocess.py:89} INFO - 03:06:27  1 of 1 OK created table model dbt_bytecode_training.dim_customers............... [SUCCESS 1 in 3.79s]
[2022-02-15 03:06:27,533] {subprocess.py:89} INFO - 03:06:27
[2022-02-15 03:06:27,535] {subprocess.py:89} INFO - 03:06:27  Finished running 1 table model in 7.64s.
[2022-02-15 03:06:27,540] {subprocess.py:89} INFO - 03:06:27
[2022-02-15 03:06:27,543] {subprocess.py:89} INFO - 03:06:27  Completed successfully
[2022-02-15 03:06:27,546] {subprocess.py:89} INFO - 03:06:27
[2022-02-15 03:06:27,548] {subprocess.py:89} INFO - 03:06:27  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 03:06:28,019] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:06:28,059] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.dim_customers, execution_date=20220215T030457, start_date=20220215T030611, end_date=20220215T030628
[2022-02-15 03:06:28,108] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:06:28,153] {local_task_job.py:264} INFO - 2 downstream tasks scheduled from follow-on schedule check
